Refactor sass lint

slifin opened the pull request.

kasbah commented:
Why did you change all the indentation to two spaces? I'd prefer 4 as it's easier to read. Using 2 spaces for JSX because of all the HTML tag nesting.

kasbah commented:
Please rebase to resolve conflicts and also squash commits so not to from 4 to 2 and then back to 4 spaces.

slifin commented:
The reasoning was that the default of scss linter setting is 2 spaces: 
https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#indentation
as does the JavaScript indentation, but the JSX solution makes sense

I have updated the linter to reflect 4 spaces and reverted code to 4 spaces for scss
